hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineManagerImpl.java

```diff
@@ -536,9 +543,9 @@ null, pipelineManager, new EventQueue(),
 
   @Test
   public void testScrubPipelines() throws Exception {
-    // No timeout for pipeline scrubber.
+    // Allocated pipelines should not be scrubbed for 50 seconds.
     conf.setTimeDuration(
-        OZONE_SCM_PIPELINE_ALLOCATED_TIMEOUT, -1,
+        OZONE_SCM_PIPELINE_ALLOCATED_TIMEOUT, 50000,
         TimeUnit.MILLISECONDS);
```

Comment:
Maybe it's just me, but I prefer `50, TimeUnit.SECONDS` to `50000, TimeUnit.MILLISECONDS` for better readability.

Reply:
Yea I agree, I just used what was there. I will change this.